src/main/resources/scripts/cleanup/non_prod_primary_and_replica_data_cleanup.sql

```diff
@@ -0,0 +1,101 @@
+--
+-- RWA-2482 cleanup old tasks from primary task database
+--
+--Example for calling the function
+--select cft_task_db.task_cleanup_between_dates_primary('2023-04-04 11:00'::TIMESTAMP,'2023-04-04 11:01'::TIMESTAMP);
+--
+
+CREATE OR REPLACE FUNCTION cft_task_db.task_cleanup_between_dates_primary(create_from TIMESTAMP,create_to TIMESTAMP)
```

Comment:
sorry, should have seen this before but should it not be `created_from` and `created_to`?

Reply:
Apologies. I missed it too. I will update the names.